src/helpers/getRandomString.test.js

```diff
@@ -0,0 +1,9 @@
+import getRandomString from './getRandomString';
+
+describe('helpers/getRandomString', () => {
+  it('should return a random string', () => {
+    const value = getRandomString();
+
```

Comment:
You could get a second string and test if they are different, to justify the test name 😉